testing-cci-datasets/auto-test-cate-datasets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+set -m
+
+# export necessary environment variables
+export EMAIL_PASSWORD="rxH6BDDy4UqSs2k"
+
+test_directory=~/projects/cate-e2e/testing-cci-datasets
+project_dirs=~/projects
+declare -a test_modes=("development" "stage" "production")
+
+git pull
+# you need to specify here, which envs should be rebuild. For stage and production this only
+# happens if a new dev or stable release has been issued. After building a new env for stage and production
+# please remember to remove them from update_envs array
+#declare -a update_envs=("development")
+declare -a update_envs=("dev_local")
```

Comment:
This is just a placeholder. Not updating development env currently due to issue with newest dask version